site/gatsby-site/content/about/index.es.mdx

```diff
@@ -41,9 +41,60 @@ La base de datos de incidentes es administrada de manera participativa por perso
\ No newline at end of file
+* **[Sean McGregor](https://seanbmcgregor.com/):** Sean McGregor fundó el proyecto de la base de datos de incidentes de IA y recientemente dejó un puesto como arquitecto de aprendizaje automático en la startup del acelerador neuronal Syntiant para poder concentrarse en la seguridad. de sistemas inteligentes a tiempo completo. El trabajo del Dr. McGregor abarca aceleradores neuronales para la inferencia de eficiencia energética, aprendizaje profundo para el habla y la heliofísica, y aprendizaje de refuerzo para la política de supresión de incendios forestales. Fuera de su trabajo remunerado, Sean organizó una serie de talleres en las principales conferencias académicas de IA sobre el tema "AI for Good" y actualmente está desarrollando un enfoque basado en incentivos para hacer que la IA sea más segura a través de auditorías y seguros. **Contribuciones:** Sean es voluntario como mantenedor de proyectos y editor del proyecto Base de datos de incidentes de IA (AIID).
+
+**Miembros sin derecho a voto**
+
+* **[Neama Dadkhahnikoo](https://www.linkedin.com/in/neama/):** Neama Dadkhahnikoo es experta en inteligencia artificial y emprendimiento, con más de 15 años de experiencia en desarrollo de tecnología en startups, no beneficios y grandes empresas. Actualmente se desempeña como gerente de productos para Vertex AI, la plataforma unificada de Google Cloud para el aprendizaje automático de extremo a extremo. Anteriormente, el Sr. Dadkhahnikoo fue director de IA y operaciones de datos en la Fundación XPRIZE, CTO de CaregiversDirect (inicio de IA para atención domiciliaria), cofundador y director de operaciones de Textpert (inicio de IA para salud mental) y consultor de inicio. Comenzó su carrera como desarrollador de software para The Boeing Company. El Sr. Dadkhahnikoo tiene un MBA de UCLA Anderson; una Maestría en Gestión de Proyectos del Stevens Institute of Technology; y una licenciatura en Matemáticas Aplicadas y Ciencias de la Computación, con especialización en Física, de UC Berkeley.
+**Contribuciones:** Neama se ofrece como voluntario para mantener organizada la Colaboración de IA responsable fuera de su rol como gerente de producto en Google.
+* **[Kit Harris](https://www.linkedin.com/in/kitharris/):** Kit dirige investigaciones de subvenciones e investiga campos prometedores en Longview Philanthropy. También escribe sobre las recomendaciones e investigaciones de Longview para filántropos clientes de Longview. Antes de centrarse en el trabajo filantrópico de alto impacto, Kit trabajó como comerciante de derivados de crédito con J.P. Morgan. Durante ese tiempo, donó la mayoría de sus ingresos a organizaciones benéficas de alto impacto. Kit tiene un título de primera clase en matemáticas de la Universidad de Oxford.
+**Contribuciones:** Kit sirve como observador de la junta, brinda asesoramiento estratégico y es el punto de contacto de RAIC en Longview.
```

Comment:
This sentence sounds weird in Spanish, especially the word `sirve`. It could be helpful to have the original text to understand the context in English